editor.planx.uk/src/@planx/components/Feedback/Public/Public.tsx

```diff
@@ -98,19 +101,26 @@ const FeedbackComponent = (
             />
           </Grid>
         </StyledToggleButtonGroup>
-        {/* </InputLabel> */}
-        <InputLabel label="Please tell us more about your experience">
-          <RichTextInput
-            name="feedback"
-            value={formik.values.feedback}
-            onChange={formik.handleChange}
+        {props.freeformQuestion && (
+          <InputLabel
+            label={
+              <ReactMarkdownOrHtml
+                source={props.freeformQuestion}
+                id={"RATING_QUESTION"}
+                openLinksOnNewTab
+              />
+            }
           />
-        </InputLabel>
+        )}
+        <RichTextInput
+          name="feedback"
+          value={formik.values.feedback}
+          placeholder="What did you think?"
+          onChange={formik.handleChange}
+        />
       </Box>
       <Typography variant="caption">
-        The information collected here isn't monitored by planning officers.
-        Don't use it to give extra information about your project or submission.
-        If you do, it cannot be used to assess your project.
+        <ReactMarkdownOrHtml source={props?.disclaimer} id={"DISCLAIMER"} />
       </Typography>
```

Comment:
nit: Rather than a `"caption"` style here we probably want to mimic the existing disclaimer styles in use on other components as those have passed a11y audits and feel a bit more Gov UK Design system-y I think - check out `PlanningConstraints` & `Result` for examples!